brightener.cpp

```diff
@@ -27,6 +27,6 @@ int ImageBrightener::BrightenWholeImage() {
 	return attenuatedPixelCount;
 }
 
-Image ImageBrightener::GetImage() {
-	return m_inputImage;
-}
+//Image& ImageBrightener::GetImage() {
```

Comment:
It's good to remove functions like `GetImage`, which are not used.
But don't leave commented lines of code in your commits. Remove the lines. If at all they are needed, they will be present in the GitHub history.
Similarly, remove all commented code from older commits as well (not only yours)